plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/UIUtils.java

```diff
@@ -1671,6 +1671,30 @@ public static IWorkbenchWindow getParentWorkbenchWindow(Control control) {
         return null;
     }
 
+    /**
+     * We try to find last visible child shell, because some Linux desktop environments are handling window modality differently
+     * see #13691
```

Comment:
Don't describe the reason _why_ this method exists. Describe what it does instead.